python/kafka/producer/kafka_producer.py

```diff
@@ -0,0 +1,21 @@
+from kafka import KafkaProducer
+from datetime import datetime
+from kafka_producer_config import from_env
+import time
+
+if __name__ == '__main__':
+    config = from_env()
+    print(config.properties)
```

Comment:
In the consumer we used `logging` instead of `print`. Let's be consistent

Reply:
print used throughout